docs/source/api/APIRef.rst

```diff
@@ -228,14 +228,14 @@ Searching for datasets locations
             :caption: Shell
 
             curl -X GET \
-            'http://api.freva.example/api/databrowser/data_search/freva/file?product=EUR-11&fs_type=swift'
+            'https://www.freva.dkrz.de/api/databrowser/data_search/freva/file?product=EUR-11&fs_type=swift'
 
         .. code-tab:: python
             :caption: Python
 
             import requests
             response = requests.get(
-                "http://api.freva.example/api/databrowser/data_search/freva/file",
+                "https://www.freva.dkrz.de/api/databrowser/data_search/freva/file",
                 pramas={"product": "EUR-11", "fs_type": "swift"}
```

Comment:
`params` has to be `params`? i think several times happened here and in Freva-nextgen as well: https://github.com/FREVA-CLINT/freva-nextgen/blob/main/docs/source/databrowser/APIRef.rst